src/hotspot/share/utilities/vmError.cpp

```diff
@@ -335,11 +335,11 @@ static frame next_frame(frame fr, Thread* t) {
     } else {
       // is_first_C_frame() does only simple checks for frame pointer,
       // it will pass if java compiled code has a pointer in EBP.
-      if (os::is_first_C_frame(&fr)) return invalid;
+      if (os::is_first_C_frame(&fr, t)) return invalid;
```

Comment:
Is the comment still accurate?

Reply:
I think so? But maybe removing the second line would be helpful.

Reply:
But are the checks still "simple"?

Reply:
After the change proposed by Thomas: I think so, it still only checks the pointer value and safefetches the value of the stack pointer, ... to check whether they are valid.